### PR TITLE
PSMDB-1404 Release notes 5.0.24-21

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   .github/workflows/main.yml
	new file:   docs/release_notes/5.0.24-21.md
	modified:   docs/release_notes/index.md
	modified:   mkdocs-base.yml
	modified:   mkdocs-pdf.yml
	modified:   variables.yml

### CODE REVIEW

File: docs/release_notes/5.0.24-21.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+It supports MongoDB 5.0.24 protocols and drivers.
+
+## Release Highlights
+
+The bug fixes, provided by MongoDB and included in Percona Server for MongoDB, are the following:
+
+* [SERVER-50792](https://jira.mongodb.org/browse/SERVER-50792) - Improved shard key index error messages by adding detailed information about an invalid index.
+* [SERVER-77506](https://jira.mongodb.org/browse/SERVER-77506) - Fixed the issue with data and ShardVersion mismatch on sharded multi-document transactions by exposing the maxValidAfter timestamp alongside the shardVersion
+* [SERVER-81878](https://jira.mongodb.org/browse/SERVER-81878) - Improved the startupRecoveryForRestore behavior by allowing writes in read-only mode from ident reaper
```
Comment (igorsol) [inline docs/release_notes/5.0.24-21.md]: SERVER-81878 should be excluded from the list because we reverted this fix (see SERVER-84432)

**Review by igorsol — CHANGES_REQUESTED** [review]

File: docs/release_notes/5.0.24-21.md (hunk 1 above)
Comment (nastena1606) [reply docs/release_notes/5.0.24-21.md]: Excluded

**Review by igorsol — APPROVED** [review]
LGTM

File: mkdocs-base.yml [hunk 2]
```diff
@@ -125,7 +125,7 @@ plugins:
   with-pdf: # https://github.com/orzih/mkdocs-with-pdf
           output_path: '_pdf/PerconaServerforMongoDB-5.0.pdf'
           cover_title: 'Server for MongoDB Documentation 5.0'
-          cover_subtitle: 5.0.23-20 (December 21, 2023)
+          cover_subtitle: 5.0.24-21 (February 2, 2024)
```
Comment (sandraromanchenko) [inline mkdocs-base.yml]:
```suggestion
          cover_subtitle: 5.0.24-21 (February 1, 2024)
```

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]

**Review by sandraromanchenko — APPROVED** [review]

**Pull request merged**